.github/workflows/test.yml

```diff
@@ -0,0 +1,148 @@
+# GitHub Actions CI definition for TensorBoard.
+#
+# YAML schema for GitHub Actions:
+# https://help.github.com/en/actions/automating-your-workflow-with-github-actions/workflow-syntax-for-github-actions
+#
+# Helpful YAML parser to clarify YAML syntax:
+# https://yaml-online-parser.appspot.com/
+
+name: CI
+
+on:
+  push: {}
+  pull_request: {}
+  schedule:
+    # 16:00 UTC is noon in Pacific standard time (UTC-8).
```

Comment:
These are the automatic builds on master, right?  Would it be worth doing these earlier in the day so that they're less likely to collide with real PR builds (and get us a fresh cache if we get caching working)?

Looking at recent `tf-nightly`  they all seem to be published between 2am and 3am pacific, so running this job at 5am should be fine (and would pick up our `tb-nightly` then too for consistency, not that it should matter).

Reply:
Sure, that sounds good to me. Will change.

Reply:
Apparently my comment was wrong anyway because I am incapable of
performing arithmetic. This was 08:00 PT and is now 05:00 PT/08:00 ET.